js/index.js

```diff
@@ -40,3 +40,119 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const imgID =
+document.getElementById("logo-img")
+// console.log(imgID)
+
+const Idimg =
+document.getElementById("cta-img")
+// console.log(Idimg)
+
+const navStyles = 
+document.querySelectorAll("nav a")
+
+navStyles.forEach(element => {
+  element.stlye.color = 'green'
+});
+
+const newCont = 
+document.querySelector('header')
+
+// nav
+
+const NewNav =
+document.createElement("a");
+
+NewNav.textContent =   {
+  "nav-item-1": "Services",
+  "nav-item-2": "Product",
+  "nav-item-3": "Vision",
+  "nav-item-4": "Features",
+  "nav-item-5": "About",
+  "nav-item-6": "Contact",
+  "img-src": "img/logo.png"
+}
+
+const nav = 
+document.querySelector("nav")
+nav.appendChild(header)
+
+// const NewCTA = 
+// documentquerySelector('.cta')
+
+// cta
+
+const newH1 =
+document.createElement("h1");
+newH1.textContent =  "DOM Is Awesome";
+
+const newButton = 
+document.createElement("button")
+newButton.textContent = "Get Started";
+
+const imgID =
+document.getElementById("logo-img")
+// console.log(imgID)
+
+// maincontent
+
+const FeatH4 = 
+document.createElement("features-h4")
+FeatH4.textContent = "Features";
+
+const H4Cont = 
+document.createElement("features-content")
+H4Cont.textContent =  "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
+
+const AH4 = 
+document.createElement("about-h4")
+AH4.textContent = ("About");
+
+const AboutCont = ("about-content")
+document.createElement = "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
+
+const midImg = 
```

Comment:
`midImg` is `null`
make sure to query the right element
```
<img class="middle-img" id="middle-img" src="" alt="Image of code snippets across the screen">
```
the query should be:
```
const midImg = document.querySelector(".middle-img");
```
remember to put in the `.` to indicate that it is a class